tox.ini

```diff
@@ -31,7 +32,6 @@ commands =
 
 [testenv:style]
 basepython = python3
-sitepackages = False
```

Comment:
And this? I suspect that we can flip sitepackages=False at the top right?

Reply:
This was to allow the test to use the Ubuntu-provided wxPython, which would not be in the tox-created virtualenv otherwise.